src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -97,20 +115,20 @@ WMS_Provider.prototype.getColorTexture = function getColorTexture(tile, layer, t
         }
         extent = parentAtLevel.extent;
         targetLevel = parentAtLevel.level;
+        const pow = 1 / (coordWMTS.zoom - targetLevel);
+        const row = Math.floor(Math.pow(coordWMTS.row, pow));
```

Comment:
`row` and `column` stuff seems out of place as well.
Any comments to explain what is going on here?

Reply:
i rool back on that